examples/Client/DistributedLockApi/README.md

```diff
@@ -0,0 +1,78 @@
+# Example - Distributed Lock APIs
+
+This example demonstrates the Distributed Lock APIs in Dapr.
+It demonstrates the following APIs:
+- **Distributed Lock**: Try Lock
+- **Distributed Lock**: Unlock
+
+> **Note:** Make sure to use the latest proto bindings
+
+## Prerequisites
+
+- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
+- [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+- [Dapr .NET SDK](https://docs.dapr.io/developing-applications/sdks/dotnet/)
+- Initialize Dapr environment by pulling the code from master branch of [Dapr](https://github.com/dapr/dapr)
+
+## Overview
```

Comment:
Can we provide a little more context in this overview? When would a developer need to ouse this API? At the very least, can we link back to the building block overview page that describes this API in more detail?

Reply:
Hi Nick! I am waiting for the distributed lock api overview page. Once it is merged on the docs side, I will go ahead and add that link here.